spec/support/fixtures.rb

```diff
@@ -0,0 +1,7 @@
+def fixture_file(relative_path)
+  File.read(fixture_file_path(relative_path))
```

Comment:
There's already a default RSpec method called `file_fixture`.
You could do something like `file_fixture('products/dummy_products.json').read`.

Reply:
good catch, updated